examples/FIL_streamz_pipeline.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<p>It's a good idea to include some conclusions here and possible ideas for how things can be expanded in the future.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>